src/views/picking.js

```diff
@@ -907,6 +907,21 @@ export class shopping {
     return substr_arr[1]+"/"+substr_arr[0]
   }
 
+  formatBin(rawStr){
+    if (!rawStr)
+      return null;
+
+    if (rawStr.length == 3)
+      return rawStr;
+
+    var newBinFormat = rawStr.match(/^(\d{1,2})([A-Z])(\d{2})(\d)$/);
+    if (newBinFormat) {
+      return newBinFormat[1] + '-' + newBinFormat[2] + '-' + newBinFormat[3] + '-' + newBinFormat[4];
```

Comment:
How are you formatting this. I was expecting
11-A69-0 but seems you might be putting another dash between the A-69?